assets/schema_input.json

```diff
@@ -10,10 +10,15 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "meta": ["id"],
+                "meta": ["irida_id"],
                 "unique": true,
                 "errorMessage": "Sample name must be provided and cannot contain spaces"
             },
+            "sample_name": {
+                "type": "string",
+                "meta": ["id"],
+                "errorMessage": "Sample name must be provided"
+            },
```

Comment:
Should `sample_name` have the same `pattern` as `sample`? Does it make sense to enforce unique here too? Otherwise, is there potential for two samples with the same `meta.id` to clobber each other?

Reply:
The intention behind omitting `unique = true` from the `input_schema.json` was to prevent the pipeline from failing immediately due to duplicate samples. Instead, the pipeline uses logic to generate unique names by combining `[id]` and `[irida_id]`, allowing it to complete successfully.

Reply:
Should we modify the error message of "sample_name" so that it doesn't say "must be provided" because it is no longer a required for the input? Maybe "If no sample name given will default to sample".

Reply:
Great catch - Thank you Steven! Changed here: [f56b9e7](https://github.com/phac-nml/speciesabundance/pull/24/commits/f56b9e7752996978af5b43084b9432e7684ec962)